Libplanet/Net/Protocols/RoutingTable.cs

```diff
@@ -91,7 +91,9 @@ public IEnumerable<BoundPeer> PeersToBroadcast(Address? except)
             if (count < _minPeersToBroadcast)
             {
                 peers.AddRange(Peers
-                    .Where(peer => except is null || !peer.Address.Equals(except.Value))
+                    .Where(peer =>
+                        !peers.Contains(peer) &&
```

Comment:
It doesn't seem related to adding breadcrumb. Is it intended? (if so, how about writing about it to changelogs?)

Reply:
First time I thought this should be in a separated PR, but without this new test case fails. I'll add this change to the changelog.